### DISCUSSION TITLE
X00TD-crdroidandroid-13.0-strongreasons

### DISCUSSION THREAD

**Discussion by strongreasons** [ROM-builders/temporary#22691] [General]
Hi Apon77,
I want to add `X00TD-crdroidandroid-13.0-strongreasons` to this organization. [Here](https://github.com/strongreasons/temporary/tree/X00TD-crdroidandroid-13.0-strongreasons) is the link to the repository which I want to add. Please review it and help me to contribute to the custom Aosp Open Source Project.
Thank you.

**Comment by Apon77** [comment]
give telegram id

**Reply by strongreasons** [reply]
https://t.me/znieeb

**Reply by Apon77** [reply]
Github and telegram names does not matches

**Reply by strongreasons** [reply]
oh sorry, I didn't think it should be the same
https://t.me/strongreasons
